citi-onboarding/virtu-onepage
# Hotfix/testimonial assets

**What o did:**
- I resolved the problem with the testimonial's assets.
- In style, I made an inheritance system with the carousel elements.
- Now if you add another testimonial card in Keystone, the new ball will be added to the carousel and the container of the navigation elements will expand according to the number of balls, without breaking.

**How to test:**
1- First go to client and server folders and give a npm install or yarn install in both to install the project dependencies.
2- Then start React server in client folder by giving a npm start or yarn start. Also go to server folder and start keystone server with the command: 'node index.js'.
3- Add a New testimonial card on keystone and see If the bullets system Is working fine. :)
